researcher_ids/README.md

```diff
@@ -26,6 +26,13 @@ Data access authorization information is encoded as Passports and transmitted ac
 
 Organizations and individuals issuing and using Passport Visas must adhere to corresponding data access and data protection policies and regulations, with respect to both the data to which Passport Visas may be used to gain access, and to the personal data of the individual to which Passport Visas apply.
 
+In addition to the further details below, there are two available recorded webinars to help get you started:
+1) A high level overview of the GA4GH Passports concept and benefits
+[![](http://img.youtube.com/vi/l5Cu76NQyUY/0.jpg)](http://www.youtube.com/watch?v=l5Cu76NQyUY "")
```

Comment:
Somehow this video starts 7 minutes in for me. Maybe add "&t=7" to the youtube URL to start at the beginning?

Reply:
It shows for me (on both of them) at the start of the video.  Is it possible that you had watched part way through the video before and this brings you to wear you left off?